charts/fybrik/files/taxonomy/taxonomy.json

```diff
@@ -338,6 +357,19 @@
         "topic_name"
       ]
     },
+    "localfile": {
+      "type": "object",
+      "description": "Connection information for accessing data in local filesystem",
+      "properties": {
+        "destination_path": {
+          "type": "string",
+          "description": "Path to the directory where the data is stored"
+        }
+      },
+      "required": [
+        "destination_path"
```

Comment:
What does this path include? Can it be split into more details, to be able to generate a connection based on pre-exisitng information for storage allocation?